src/host/renderData.cpp

```diff
@@ -329,7 +325,7 @@ const std::vector<size_t> RenderData::GetPatternId(const til::point /*location*/
 }
 #pragma endregion
 
-#pragma region IUiaData
+#pragma region IRenderData

```

Comment:
Remove these `#pragma ` lines altogether. The `endregion` doesn't have a matching start region, and the `#pragma region` doesn't have a matching `endregion` haha